http_check/datadog_checks/http_check/http_check.py

```diff
@@ -74,10 +74,7 @@ def send_status_down(loginfo, down_msg):
 
         # Store tags in a temporary list so that we don't modify the global tags data structure
         tags_list = list(tags)
-        url = instance.get('url', None)
-        if url is not None:
-            url = ensure_unicode(url)
```

Comment:
Do we still want to ensure_unicode on addr?

Reply:
https://github.com/DataDog/integrations-core/blob/04e61c5ce4c3ae30d8a867ab269cb579cb1a01f3/http_check/datadog_checks/http_check/config.py#L44-L46

Reply:
Ahh, missed that its done in config.py (we use addr here and url there)